src/Custom/Responses/ResponseResult.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.Json;
+
+namespace OpenAI.Responses;
+
+// CUSTOM:
+// - Added Experimental attribute.
+// - Renamed.
+[CodeGenType("Response")]
+[CodeGenSuppress("OutputText")]
+public partial class ResponseResult
+{
+    // CUSTOM: Renamed.
+    [CodeGenMember("Background")]
+    public bool? BackgroundModeEnabled { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("User")]
+    public string EndUserId { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("Reasoning")]
+    public ResponseReasoningOptions ReasoningOptions { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("MaxOutputTokens")]
+    public int? MaxOutputTokenCount { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("MaxToolCalls")]
+    public int? MaxToolCallCount { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("Text")]
+    public ResponseTextOptions TextOptions { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("Truncation")]
+    public ResponseTruncationMode? TruncationMode { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("IncompleteDetails")]
+    public ResponseIncompleteStatusDetails IncompleteStatusDetails { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("Output")]
+    public IList<ResponseItem> OutputItems { get; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("ParallelToolCalls")]
+    public bool ParallelToolCallsEnabled { get; set; }
+
+    // CUSTOM: Using convenience type.
+    [CodeGenMember("ToolChoice")]
+    public ResponseToolChoice ToolChoice { get; set; }
+
+    // CUSTOM: Use a plain string.
+    [CodeGenMember("Model")]
+    public string Model { get; set; }
+
+    // CUSTOM: Renamed.
+    [CodeGenMember("TopLogprobs")]
+    public int? TopLogProbabilityCount { get; set; }
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public string Object { get; set; } = "response";
+
+    [CodeGenMember("Metadata")]
+    public IDictionary<string, string> Metadata { get; }
+
+    [CodeGenMember("Temperature")]
+    public float? Temperature { get; set; }
+
+    [CodeGenMember("TopP")]
+    public float? TopP { get; set; }
+
+    [CodeGenMember("SafetyIdentifier")]
+    public string SafetyIdentifier { get; set; }
+
+    [CodeGenMember("ServiceTier")]
+    public ResponseServiceTier? ServiceTier { get; set; }
+
+    [CodeGenMember("PreviousResponseId")]
+    public string PreviousResponseId { get; set; }
+
+    [CodeGenMember("Instructions")]
+    public string Instructions { get; set; }
+
+    [CodeGenMember("Tools")]
+    public IList<ResponseTool> Tools { get; set; }
```

Comment:
I think this one is unchanged (minus the setter, which needs to be removed), so we can remove it from custom code.